app.rb

```diff
@@ -43,151 +43,79 @@ def create_person
         create_teacher
         break
       else
-        puts 'Invalid input.Please Choose 1 for student 2 for teacher.'
+        puts 'Invalid input. Please choose 1 for student or 2 for teacher.'
```

Comment:
- [ ] I am not able to see where you wrote your tests please add them and make sure you tested all classes
<img width="554" alt="Screenshot 2023-08-15 at 21 23 22" src="https://github.com/derrick1451/schools_library/assets/44745855/b46c2041-70ee-4a3a-8088-96b7aa09082a">